src/AffectedUserFlow/CodeInput/CodeInputForm.tsx

```diff
@@ -214,18 +200,13 @@ const LoadingIndicator = () => {
 const indicatorWidth = 120
 
 const styles = StyleSheet.create({
-  backgroundImage: {
-    width: "100%",
-    height: "100%",
-    resizeMode: "cover",
-    backgroundColor: Colors.faintGray,
-  },
   container: {
     height: "100%",
     justifyContent: "space-between",
     paddingHorizontal: Spacing.medium,
     paddingTop: Layout.oneTenthHeight,
```

Comment:
Should we just be using a `Spacing` var here? IMO we should only use `Layout` vars if necessary for some reason.

Reply:
We could do, but there is no spacing var that is around 1/10th the height of the screens at this point and I dont want to get too into the weeds fiddling with layout on this pr.